book/05-distributed-git/sections/contributing.asc

```diff
@@ -270,8 +270,10 @@ Switched to branch 'master'
 Your branch is behind 'origin/master' by 2 commits, and can be fast-forwarded.
 ----
 
-Jessica can merge either `origin/master` or `issue54` first -- they're both upstream, so the order doesn't matter.
-The end snapshot should be identical no matter which order she chooses; only the history will be different.
+Jessica can merge either `origin/master` or `issue54` first -- the order doesn't matter.
+The end snapshot will be identical no matter which order she chooses; only the history will be different.
+However let's mention the fact in this case to be worth to merge `issue54` first as it will be just fast-forward -
+no new commit is created - resulting in version tree bit more clearer.
```

Comment:
More grammatical updates:

> In this case, she chooses to merge `issue54` first, since it will be a fast-forward merge, which will result in a simpler history.